data-runtime/src/main/java/io/micronaut/data/runtime/operations/internal/query/BindableParametersStoredQuery.java

```diff
@@ -110,5 +111,8 @@ default int currentIndex() {
             return -1;
         }
 
+        default Dialect getDialect() {
```

Comment:
You don't need to add getDialect method, all the information is present in the parent classes